tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL2807Test.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+final class DBAL2807Test extends DbalFunctionalTestCase
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function setUp()
+    {
+        parent::setUp();
+
+        if (!in_array($this->getPlatform()->getName(), ['mysql']))
+        {
+            $this->markTestSkipped('Restricted to MySQL.');
+
+            return;
```

Comment:
This is unnecessary.

Reply:
@arnegroskurth I see. @Majkl578 what if we disable this rule? It's often so annoying and forces write cumbersome code. In this case it could probably be:
```php
if (in_array(/* ... */)) {
    return;
}

$this->markTestSkipped();
```

Reply:
@morozov Please just change the code in the way you want it to be yourself. I opened up the MR two months (!!) ago and seven out of eleven commits are dedicated to code-styling and such. I know that I can use phpcs locally and I did. But if you still have issues with the code after your own style-definition said it would be OK then I really cannot help it :-1:

Reply:
> Please just change the code in the way you want it to be yourself.

I just did.

> But if you still have issues with the code after your own style-definition said it would be OK then I really cannot help it -1

Our coding standard has issues as well as any software does.